scripts/ebs-warmer.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Source: https://gist.github.com/muhqu/3293988
+# Ref: https://forums.aws.amazon.com/message.jspa?messageID=198413
+
+if [ -z "$1" ]; then
+    echo "Usage: sudo $0 /dev/sdh1"
+    exit 1;
+fi
+
+dd if=$1 of=/dev/null & pid=$!
```

Comment:
The [AWS docs recommend](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-initialize.html) adding a blocksize argument at 1M `bs=1M`